pkg/builtinchecks/yamls/dnsconfig-options.yaml

```diff
@@ -0,0 +1,12 @@
+name: "dnsconfig-options"
```

Comment:
Is this implementation too `ndots` specific? In other words, does the solution as written provide a mechanism to check other aspects of `dnsConfig` like `nameservers` and `searches`?

Reply:
Current implementation supports matching in `options` field, such as `ndots`, `timeout`, `attempts`, etc. A full list [here under options section](https://linux.die.net/man/5/resolv.conf). `nameservers` and `searches` are at the same level as `options`, so not supported currently.